### PR TITLE
Feat/gameteam create from csv spike

### LINKED ISSUE #11: Create dummy files (dummy_games and dummy_gameteams) to work with for testing

### LINKED ISSUE #19: Add self.create_from_csv method to GameTeam class

### LINKED ISSUE #20: Create initial tests for gameteam_spec

### PR DESCRIPTION
**Pull request by wallacebilly1**
Created dummy files for game_teams and games data sources; Created tests and methods for initialize and create_from_csv for GameTeam class; added .gitignore file.

### CODE REVIEW

File: spec/gameteam_spec.rb [hunk 1]
```diff
@@ -1,5 +1,44 @@
-require './spec_helper'
+require './spec/spec_helper'
```
Comment (wallacebilly1) [inline spec/gameteam_spec.rb:1]: Don't need the "./spec/" portion.

File: lib/gameteam.rb [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+  attr_reader :game_id,
+              :team_id,
+              :hoa,
+              :result,
+              :head_coach,
+              :goals,
+              :shots,
+              :tackles
+
+  def initialize(gameteam_data)
```
Comment (wallacebilly1) [inline lib/gameteam.rb:13]: change all "gameteam" to "game_team"

File: lib/gameteam.rb [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+    @shots = gameteam_data[:shots]
+    @tackles = gameteam_data[:tackles]
+  end
+
+  def self.create_from_csv(game_teams_path)
+    game_teams = []
+    CSV.foreach(game_teams_path, headers: true, converters: :all) do |row|
+      gameteam_data = { game_id: row["game_id"],
+        team_id: row["team_id"],
+        hoa: row["HoA"],
```
Comment (wallacebilly1) [inline lib/gameteam.rb:29]: Make sure that "HoA" works properly

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+echo coverage >> .gitignore
```
Comment (wallacebilly1) [inline .gitignore:1]: Add gitignore for gemfile.lock

**Comment by AuroraTuring** [comment]
Looks great, thanks for the notes about implementing changes.

**Pull request merged**